### ISSUE TITLE
What is installed where ?

### ISSUE THREAD

**Issue by JoopStringer** [sirkirby/dropzone-nuget#9]
Hi there,

If I use the NuGet package in Visual studio I wonder where the scripts are installed.
I can't find seem to find them.

Thanks.

**Comment by hnnweb** [comment]
Unfortunately this package seems dead.

**Comment by sirkirby** [comment]
Yes, that's fair. I'm no longer actively maintaining it. I'm happy to accept a PR and get it published though.

Chris

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐
